src/Microsoft.Data.Analysis/DataFrame.cs

```diff
@@ -205,36 +205,52 @@ private DataFrame Clone(DataFrameColumn mapIndices = null, bool invertMapIndices
         }
 
         /// <summary>
-        /// Generates descriptive statistics that summarize each numeric column
+        /// Generates a concise summary of each column in the DataFrame
         /// </summary>
-        public DataFrame Description()
+        public DataFrame Info()
         {
             DataFrame ret = new DataFrame();
-            if (Columns.Count == 0)
-                return ret;
-            int i = 0;
-            while (!Columns[i].HasDescription())
+
+            for (int i = 0; i < Columns.Count; i++)
             {
-                i++;
+                DataFrameColumn column = Columns[i];
+                if (i == 0)
+                {
+                    StringDataFrameColumn strColumn = new StringDataFrameColumn("Info", 2);
+                    strColumn[0] = "DataType";
+                    strColumn[1] = Strings.DescriptionMethodLength;
+                    ret.Columns.Add(strColumn);
+                }
+                column.Info(ret);
             }
-            ret = Columns[i].Description();
-            i++;
-            for (; i < Columns.Count; i++)
+            return ret;
+        }
+
+        /// <summary>
+        /// Generates descriptive statistics that summarize each numeric column
+        /// </summary>
+        public DataFrame Description()
+        {
+            DataFrame ret = new DataFrame();
+
+            bool firstDescriptionColumn = true;
```

Comment:
(minor) why the difference in patterns here between Description and Info? Above you use a `for` loop and check for `i==0`. But here you use a `foreach` and keep track of a `firstDescriptionColumn` bool?

Reply:
Because I wrote them on different days :). It doesn't really mean anything. If I'd written `Description` first and then `Info`, I probably would've used the same pattern.

Reply:
Fixed it with the latest commit